src/qtAliceVisionImageIO/QtAliceVisionImageIOHandler.cpp

```diff
@@ -141,6 +141,7 @@ bool QtAliceVisionImageIOHandler::read(QImage *image)
 bool QtAliceVisionImageIOHandler::write(const QImage &image)
 {
     // TODO
+    (void)image;  // Remove "unused parameter" warning
```

Comment:
just remove the parameter name from the signature above:
```c++
bool QtAliceVisionImageIOHandler::write(const QImage&)
```

Reply:
Done.